src/composables/Project.js

```diff
@@ -810,7 +810,7 @@ export async function getAllModels(projectId) {
         id: `diagram_${index}`,
         plugin: plugin.data.name,
         path,
-        tags: plugin.configuration.tags,
+        tags: plugin.configuration.tags.filter(({ type }) => type === 'category'),
```

Comment:
why ?

Reply:
remove it

Reply:
Because otherwise it will also get the `Language` tags, which is not what we want in the `DiagramTable`.